README.md

```diff
@@ -200,11 +266,143 @@ df <- read.df(table_path, "deltaSharing")
 
 You can try this by running our [examples](examples/README.md) with the open, example Delta Sharing Server.
 
+### CDF 
+Starting from release 0.5.0, querying [Change Data Feed](https://docs.databricks.com/delta/delta-change-data-feed.html) is supported with delta sharing.
+The provider can turn on the CDF on the original delta table, and share it in delta sharing. Once CDF is shared, the recipient can query
+CDF of a delta sharing table in a similar way of a delta table:
```

Comment:
in a similar way of a delta table -> similar to CDF of a delta table.

Reply:
updated.